### ISSUE TITLE
Feature

### ISSUE THREAD

**Issue by bearcatsandor** [Jas-SinghFSU/HyprPanel#438]
I would love so have the option to display the playlist of the current player in the media module.  

`https://specifications.freedesktop.org/mpris-spec/latest/Track_List_Interface.html`

**Comment by rubiin** [comment]
By display , you , mean the playlist name or the contents of the playlist. Name should be straight forward however, the later one will be tedious

**Comment by bearcatsandor** [comment]
The contents of the playlist. If it's tedious, nevermind.

**Comment by rubiin** [comment]
Yeah. Its tedious in the UI part however I will add it to my todo list with a lesser priority for now :)

**Comment by bearcatsandor** [comment]
You rock!

**Comment by rubiin** [comment]
@bearcatsandor suggestions are welcome on how the UI would look like for the playlist

**Comment by bearcatsandor** [comment]
I was thinking of something very simple. You'd only need: list number, track-artist, album, title, time.  Then a scrollbar. The notifications display could be a model.

1 Gojira - Fortitude - Into the Storm 5:02
2  Frank Sinatra - '57 in Concert - Just One of those Things 3:18
.
.
43 Sara K. - Live In Concert - Vincent  5:21
